Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Python 3.8
+FROM python:3.9
+
+EXPOSE 8501
+
+# SHELL ["/bin/bash", "--login", "-c"]
+
+LABEL version="0.1.0"
+
+LABEL description="An automated text-mining tool written in Python to measure \
+    the technical responsibility of students in computer science courses, being \
+    used to analyze students' markdown reflection documents and five questions \
+    survey based on Natural Language Processing in the Department of Computer \
+    Science at Allegheny College."
+
+LABEL maintainer="Group 5"
+
+# ENV LANG C.UTF-8
+# ENV LANGUAGE C.UTF-8
+# # ENV LC_ALL C.UTF-8
+# RUN groupadd -r student && \
+#     useradd -m -r -g student -d /home/student -s /usr/sbin/nologin -c "student User" student && \
+#     mkdir -p /home/student/workdir && \
+#     chown -R student:student /home/student
+# COPY . /home/student/GATORMINER/
```

Comment:
Again, similar to above. If this commented out code is not being used, it should be removed to keep things clean